yajsapi/executor/index.ts

```diff
@@ -194,12 +202,13 @@ export class Executor {
     driver,
     network,
```

Comment:
I'm aware this would change the public API, but maybe we should update the parameter names here as well (like we did for internal `Executor` fields)?

Reply:
I added new param names and left the old ones with a deprecation warning